parsec/core/gui/claim_user_widget.py

```diff
@@ -532,31 +532,38 @@ def __init__(self, jobs_ctx, config, addr):
         self.retrieve_info_success.connect(self._on_retrieve_info_success)
         self.retrieve_info_error.connect(self._on_retrieve_info_error)
         self.claimer = Claimer()
+        self._count__ = 0
         self._run_claimer()
 
     def _run_claimer(self):
+        print("_run_claimer", self._count__)
```

Comment:
Remove the print and the count variable.

Reply:
:+1: